x-pack/plugins/lens/public/indexpattern_datasource/indexpattern.test.ts

```diff
@@ -383,7 +383,16 @@ describe('IndexPattern Data Source', () => {
                             true,
                           ],
                           "extended_bounds": Array [
-                            "{}",
+                            Object {
+                              "chain": Array [
+                                Object {
+                                  "arguments": Object {},
+                                  "function": "extendedBounds",
+                                  "type": "function",
+                                },
+                              ],
+                              "type": "expression",
```

Comment:
Since the `extended_bounds` was always empty before, why is it now a required argument? Seems like it's adding extra complexity.

Reply:
Those are equivalent. We cannot pass an object in the expressions language without serializing that. In this PR, we are decoupling the deserialization into dedicated expressions functions.